client/my-sites/post-selector/pagination.jsx

```diff
@@ -0,0 +1,64 @@
+/**
```

Comment:
@aduth I mean removing the PostSelectorPagination child and making QueryPosts + Selector direct children of PostSelector.